proto/lib/protocol/bep.proto

```diff
@@ -164,6 +166,7 @@ message Request {
     bytes  hash           = 6;
     bool   from_temporary = 7;
     uint32 weak_hash      = 8;
+    int32  block_no       = 9;
```

Comment:
So this is for us to make it easier to find what's the real offset in the unencrypted file that we're after, but that's a deterministic function anyway, so I am not really use we need it?

It's going to be Offset/EncryptedBlockSize (where EncryptedBlockSize is deterministici as well)?

Reply:
It isn't really because the block size isn't deterministic, it depends on the file size now and the file size in the past, as we may have hashed it once drifted away from the original size.

Reply:
But we have the size given to us, so surely we can work out what block it's referring to based on it's size?

Reply:
I mean imagine a world where syncthing is content addressed, and all requests are served by hash. I think that would still work without the need of block no et al.